### PR TITLE
Run scalafmt with the new version

### PR DESCRIPTION
**Pull request by aeons**
and remove the old, unused plugin.

### CODE REVIEW

**Comment by aeons** [comment]
This got bigger than it looked in the beginning. I'll see if we can configure scalafmt to do something close to what we had before.

**Comment by aeons** [comment]
I changed the config to make the diff smaller.

File: http/src/main/scala/org/http4s/blaze/http/http2/bits.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@ private[http2] object bits {
 
   object Masks {
     val INT31: Int = 0x7fffffff
-    val INT32: Long = 0xffffffffL
+    val INT32: Long = 0XFFFFFFFFL
```
Comment (rossabaker) [inline http/src/main/scala/org/http4s/blaze/http/http2/bits.scala]: It's curious it doesn't capitalize the `Int` literals, too.

File: core/src/main/scala/org/http4s/blaze/channel/nio2/ClientChannelFactory.scala [hunk 2]
```diff
@@ -48,7 +48,9 @@ final class ClientChannelFactory(
             s"An attempt to establish connection with $remoteAddress timed out after $connectTimeout.")
           val finishedWithTimeout = p.tryFailure(exception)
           if (finishedWithTimeout) {
-            try { ch.close() } catch { case NonFatal(_) => /* we don't care */ }
+            try {
+              ch.close()
+            } catch { case NonFatal(_) => /* we don't care */ }
```
Comment (rossabaker) [inline core/src/main/scala/org/http4s/blaze/channel/nio2/ClientChannelFactory.scala]: Do we know why it's more aggressively wrapping things now?  It's fine, but I shudder to think how large the diff is going to be on http4s core.

File: http/src/main/scala/org/http4s/blaze/http/http2/bits.scala (hunk 1 above)
Comment (aeons) [reply http/src/main/scala/org/http4s/blaze/http/http2/bits.scala]: Yea, that's weird.

File: core/src/main/scala/org/http4s/blaze/channel/nio2/ClientChannelFactory.scala (hunk 2 above)
Comment (aeons) [reply core/src/main/scala/org/http4s/blaze/channel/nio2/ClientChannelFactory.scala]: I'll see if I can get the config to leave those alone.

**Comment by aeons** [comment]
Haven't really had any luck with the oneline lambdas.

File: core/src/main/scala/org/http4s/blaze/channel/nio2/ClientChannelFactory.scala (hunk 2 above)
Comment (aeons) [reply core/src/main/scala/org/http4s/blaze/channel/nio2/ClientChannelFactory.scala]: https://github.com/scalameta/scalafmt/issues/1437

Comment (aeons) [reply core/src/main/scala/org/http4s/blaze/channel/nio2/ClientChannelFactory.scala]: Yeah, that link was meant for the other comment.

**Pull request merged**